### PR TITLE
fix(*): Updated links to tutorials

### PR DESCRIPTION
**Pull request by qbzzt**
[Based on Discord](https://discord.com/channels/667044843901681675/921343563596439602/987738042485862440)

### CODE REVIEW

File: src/docs/guides/cex-dev.md [hunk 1]
```diff
@@ -49,10 +49,7 @@ Most of the cost of an Optimism transaction is not the gas consumed by the trans
...
-
-* [For ETH](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/eth-deposit)
-* [For ERC-20 tokens](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/l1-l2-deposit-withdrawal)
+As a centralized exchange, there will be times that withdrawals of ETH or an ERC-20 token on a specific chain exceed deposits and you need to transfer assets. To do that you use a bridge or a gateway. We have a [standard gateway](https://gateway.optimism.io/) that receives assets on L1 (Ethereum mainnet), and mints the equivalent asset on L2 (which can be Optimism or a different solution). Eventually, when a user (possibly a different one) wants to withdraw the assets back to L1, the bridge burns the asset on L2 and releases it to the user on L1. If you want to use this gateway automatically, [follow this tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/cross-dom-bridge).
```
Comment (Nicca42) [inline src/docs/guides/cex-dev.md:52]: You should also add something here before eventually (on a new line) about the standard ERC20 bridge, as in the first sentence you mention it but you do not mention it again. Something like: 

> s a centralized exchange, there will be times that withdrawals of ETH or an ERC-20 token on a specific chain exceed deposits and you need to transfer assets. To do that you use a bridge or a gateway. We have a [standard gateway](https://gateway.optimism.io/) that receives assets on L1 (Ethereum mainnet), and mints the equivalent asset on L2 (which can be Optimism or a different solution). 

> For bridging we have the [Standard ERC20 bridge](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/standard-bridge-standard-token), as well as the [Custom ERC20 bridge](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/standard-bridge-custom-token). 

> Eventually, ...

File: src/docs/developers/tutorials.md [hunk 2]
```diff
@@ -13,11 +13,14 @@ lang: en-US
 ## Cross Domain
 
 * [Communication between contracts on Optimism and Ethereum](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/cross-dom-comm)
-* [Bridging Assets with the Optimism SDK](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/cross-dom-bridge)
```
[the remark this answers is not in the document]
Comment (qbzzt) [reply src/docs/developers/tutorials.md:16]: Thank you for catching this, I'll fix it. The only reason I can think of is that I don't get enough sleep ;-)

File: src/docs/guides/cex-dev.md (hunk 1 above)
Comment (qbzzt) [reply src/docs/guides/cex-dev.md:52]: The two tutorials you linked are both standard bridge, just for different token types. I added a paragraph about adding new tokens to Optimism using them.

File: src/docs/guides/smart-contract-devs.md [hunk 3]
```diff
@@ -9,9 +9,10 @@ Optimism supports [Solidity](https://docs.soliditylang.org/en/develop/) as well
...
-
-Depending on your set up you will need to set up a local testnet. All of the above tutorials have instructions on how to set up an appropriate testing environment.
+To get started we need to [get a local environment set up for testing and development](../developers/build/dev-node.md).
+Then you can follow [the getting started tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/getting-started).
+After that we have [lots of tutorials for you to follow](https://github.com/ethereum-optimism/optimism-tutorial).
+If you think we need more, you can ask for a specific topic on our [Discord](https://discord.optimism.io/) in the **#doc-requests** channel.
```
Comment (Nicca42) [inline src/docs/guides/smart-contract-devs.md:15]: This is not the right Discord link. Please use this: https://discord-gateway.optimism.io/

Also just make sure if we use the wrong one anywhere else.

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/guides/cex-dev.md [hunk 4]
```diff
@@ -49,15 +50,20 @@ Most of the cost of an Optimism transaction is not the gas consumed by the trans
...
+When a user wants to withdraw the assets back to L1, the bridge burns the asset on L2 and releases it to the user on L1. If you want to use this gateway automatically, [follow this tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/cross-dom-bridge).
 
 Note that while L1 to L2 transactions typically take minutes, L2 to L1 transaction on the gateway require [a seven day challenge period](https://help.optimism.io/hc/en-us/articles/4411895558171-Why-do-I-need-to-wait-a-week-when-moving-assets-out-of-Optimism-).
 
 Alternatively, you can use a [third party bridge](https://www.optimism.io/apps/bridges). These bridges usually rely on liquidity pools to allow for faster withdrawals and support multiple L2 chains. However, their token selection might be more limited and they may not be as decentralized as our gateway.
 
+When an ERC-20 token does not have an Optimism equivalent you can create one. 
+If there is no need for custom business logic, you can [follow the steps in this tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/standard-bridge-standard-token).
```
[the remark this answers is not in the document]
Comment (qbzzt) [reply src/docs/guides/cex-dev.md:63]: I hate relying on my memory. Do you have a linear ticket you use to track the token repo process? Or a PM who is keeping track of it?

File: src/docs/guides/smart-contract-devs.md (hunk 3 above)
Comment (qbzzt) [reply src/docs/guides/smart-contract-devs.md]: I'll open another PR for changing it anywhere I `find` it.

Comment (qbzzt) [reply src/docs/guides/smart-contract-devs.md]: Done. See #446, it should be an easy PR to approve.

File: src/docs/guides/cex-dev.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Nicca42) [reply src/docs/guides/cex-dev.md:63]: Made an issue: https://linear.app/optimism/issue/DOCS-623/update-cex-guide-with-new-token-listing-process

File: src/docs/guides/smart-contract-devs.md [hunk 5]
```diff
@@ -32,4 +33,4 @@ There are three ways you can do this:
 
 Now that your deployer address has ETH, you will need to point to the right network to deploy. You can find the required network information as well as RPC provider information in our [networks page](../useful-tools/networks.md#optimism-mainnet).
 
-If you run into any trouble don't hesitate to ask for help on our [Discord](https://discord.optimism.io/)!
\ No newline at end of file
+If you run into any trouble don't hesitate to ask for help on our [Discord](https://discord.optimism.io/)!
```
Comment (Nicca42) [inline src/docs/guides/smart-contract-devs.md:36]: Update link

**Review by Nicca42 — CHANGES_REQUESTED** [review]

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**